README.md

```diff
@@ -1,5 +1,5 @@
 # [Decanter](https://github.com/SU-SWS/decanter)
-##### Version: 7.0.0-dev
+##### Version: 7.1.1
```

Comment:
We never keep this up to date. I've been removing it from our readmes or replacing with just the major version. The specific version in package.json is good enough.

Reply:
God idea - I 100% agree 👍🏼 😄

Reply:
This has been updated to 
`##### Version: 7`